### ISSUE TITLE
Use makeBlockNow in testing

### ISSUE THREAD

**Issue by signorecello** [EYBlockchain/nightfall_3#803]
TL;DR at the end

I'll sing you a song of my people. Once upon a time, nightfall would only create new blocks once the mempool was full. That was enough for the people of old, but was not suitable for production usage. So, great human effort was required in order to produce a function that was given the glorious name of ```makeBlockNow```, which means, in ancient English, "make block now". It's function is to make a block even if the mempool is not full.

For a time, this function was not used in testing, so humanity couldn't use it to make tests run faster. This issue proposes that this function is thereby used for testing.

TL;DR: Start using ```makeBlockNow``` in tests

**Comment by Westlad** [comment]
Fixed by #875
